src/inc/conattrs.hpp

```diff
@@ -8,7 +8,7 @@ Licensed under the MIT license.
 #define BG_ATTRS (BACKGROUND_BLUE | BACKGROUND_GREEN | BACKGROUND_RED | BACKGROUND_INTENSITY)
 #define META_ATTRS (COMMON_LVB_LEADING_BYTE | COMMON_LVB_TRAILING_BYTE | COMMON_LVB_GRID_HORIZONTAL | COMMON_LVB_GRID_LVERTICAL | COMMON_LVB_GRID_RVERTICAL | COMMON_LVB_REVERSE_VIDEO | COMMON_LVB_UNDERSCORE)
 
-enum class ExtendedAttributes : BYTE
+enum class ExtendedAttributes : uint16_t

```

Comment:
This is to make `std::has_unique_object_representations_v<TextAttribute>` happy.

Reply:
Wouldn't it be better to keep this as `BYTE` & add a `uint8_t zeropad` to `TextAttribute`?

Reply:
That's a good idea! If @skyline75489 wants to he could still change it. 🙂
Personally I'd be fine merging it either way though.